### DISCUSSION TITLE
Plot 3D IPF ?

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1502] [Ask Anything]
Dear all,
    I have  3D EBSD data, which are euler angles for each point of a cuboid. I am trying to plot a 3D ipf map.
    A method I have come up with is to plot the 6 ipf maps for the 6 surfaces of the cuboid. I can plot the ipf on the XOY plane. But how can I plot them for other planes, like ZOY?
    Thanks so much!
    Brian
![image](https://user-images.githubusercontent.com/69452222/180597148-3754696e-b7df-4b69-a9b2-e9d7362d8056.png)

**Comment by kilir** [comment]
Hi,
no this is not possbile, 3D ebsd was introduced 2011 but has been removed some years later since nobody used it because there are programs dedicated fully to 3D EBSD. However, in your case you can probably construct an EBSD object with coordinates and orientations coming from specific Y-Z or Z-Y lines.
Hope this helps.
Rüdiger
